pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/EventActionDao.java

```diff
@@ -49,16 +51,22 @@
     @CreateSqlObject
     JdbiCopyAnswerEventAction getJdbiCopyAnswerEventAction();
 
+    @CreateSqlObject
+    EventActionSql getEventActionSql();
+
+    default long insertInvitationEmailNotificationAction(SendgridEmailEventActionDto sendgridTemplateDto) {
+        return insertNotificationAction(sendgridTemplateDto, NotificationType.INVITATION_EMAIL);
+    }
 
     default long insertStudyNotificationAction(SendgridEmailEventActionDto sendgridTemplateDto) {
-        return _insertNotificationAction(sendgridTemplateDto, NotificationType.STUDY_EMAIL);
+        return insertNotificationAction(sendgridTemplateDto, NotificationType.STUDY_EMAIL);
```

Comment:
Are these now entry points to the class?

Reply:
I guess so -- if you need different types of notifcation, use the appropriate methods.